h/services/group.py

```diff
@@ -144,11 +150,15 @@ def _create(self, name, userid, description, type_flags, origins=[], add_creator
         :param description: the description of the group
         :param type_flags: the type of this group
         :param origins: the list of origins that the group will be scoped to
+        :param organization: the organization to associate the group with
         :param add_creator_as_member: if the group creator should be added as a member
         """
         creator = self.user_fetcher(userid)
         scopes = [GroupScope(origin=o) for o in origins]
 
+        if organization is None:
+            organization = Organization.default(self.session)
```

Comment:
As I understand it, the default organization is always unwaveringly associated with the `hypothesis` authority, which seems to imply here that any group created through the service with a missing (`None`) organization will get associated via the default-organization relation with `hypothesis` as an organization. This may be fine; just raising it in case it isn't.

Reply:
I think instead of organization=None we should have organization='`__default__`' and be using the pubid here then using a fetcher like we do for users.